ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -153,7 +153,11 @@ then
 		then
 		fi
 
-		if meas_airspeed start -b 2
+		if ms4525_airspeed start -b 2
```

Comment:
What about leaving all the board specific start away and add them to the overall list on the bottom?